src/poliastro/czml/extract_czml.py

```diff
@@ -276,7 +277,11 @@ def add_ground_station(
                 )  # get semi-major and semi-minor axises
             else:
                 a, b = Earth.R.to(u.m).value, Earth.R_polar.to(u.m).value
-            pos = list(map(lambda x: x.value, ellipsoidal_to_cartesian(a, b, u0, v0)))
+            
+            f=1-(b/a) #Flattenning 
+            
+            pos = list(map(lambda x: x, gd2gce(a, f, u0.to_value(u.rad), v0.to_value(u.rad), 0)))
```

Comment:
I think we can remove the `map(lambda x: x` part and leave only `list(gd2gce(...))`.

Reply:
Also, this will make the current tests fail because `ellipsoidal_to_cartesian` expects **geocentric** latitude, and `gd2gce` uses **geodetic** latitude.

In case of doubt, I would say that people expect geodetic latitude. So our choice was unfortunate.

We'll have to change the test.